src/api/app/components/build_results_monitor_component.html.haml

```diff
@@ -22,13 +22,13 @@
               - results_count_per_package_and_category(package_name).each do |key, value|
                 = render partial: 'webui/shared/build_status_count_badge', locals: { category: key, count: value.to_s }
 
-      .accordion-collapse.collapse{ id: "collapse-#{package_name.gsub(':', '_')}", class: show }
+      .accordion-collapse.collapse.my-4{ id: "collapse-#{package_name.gsub(':', '_')}", class: show }
```

Comment:
I can deal with a 3, no less than that. Otherwise it would just feel _detached and unaligned_ but not really visually separated.

Generally speaking I even sometime miss a greater than 4 value (not in this case), the usage of the space is really important. We stopped with tables styles and boxes (borders, etc) so we need space even more now.

Reply:
Done with a `my-3`. I updated the screenshot in the PR description to compare.

Reply:
> I agree, 3 seems fine. What bothers me a bit is:
> 
> ![Screenshot from 2024-02-15 12-03-38](https://private-user-images.githubusercontent.com/114928900/305049540-2ccdb8ea-0d3d-4c0c-a47b-65ecf7d85448.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDc5OTU3ODEsIm5iZiI6MTcwNzk5NTQ4MSwicGF0aCI6Ii8xMTQ5Mjg5MDAvMzA1MDQ5NTQwLTJjY2RiOGVhLTBkM2QtNGMwYy1hNDdiLTY1ZWNmN2Q4NTQ0OC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMjE1JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDIxNVQxMTExMjFaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1hMjk4YzcxNTJiN2JjYTY5ZjIxOGJjMzBkMDM3ZDk4MGZlMWRiZDIxZjcyYzZkMGUwNDFmNjZmYWZkMzJhOTljJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.TcIy8rGl-SlZ1V26zTmKvVairPWyQmMWE3KU-_97_NQ)
> 
> This should match the padding on the left side of the row, and has to only apply to last architecture cell. I was the person that broke this though, so maybe I should fix it

A-ha! Good catch, it slipped under my eye-radar. Yeah, it would be nice to fix it. If you already know what to do, feel free to do it right away and push it into this PR, it still targets the same scope.